python/sanlock.c

```diff
@@ -1674,75 +1676,87 @@ initexception(void)
     return excp;
 }
 
-PyMODINIT_FUNC
-initsanlock(void)
+static int
+util_module_init(PyObject* m)
 {
-    PyObject *py_module, *sk_constant;
-
-    py_module = Py_InitModule4("sanlock",
-                sanlock_methods, pydoc_sanlock, NULL, PYTHON_API_VERSION);
-
-    if (py_module == NULL)
-        return;
+	PyObject *align = NULL, *sector = NULL;
```

Comment:
Moving cleanup to end of function means we have these NULL objects available to entire function, instead of the relevant parts of the code.